### PR TITLE
Call for mentors

### PR DESCRIPTION
**Pull request by yabellini**
Blog post for Call for Mentors.  Was reviewed by Camille and Lou.
The idea is to publish today or tomorrow, so we can include in the newsletter.

### CODE REVIEW

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+title: rOpenSci seeks Mentors
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:2]:
```suggestion
title: Become a Mentor for rOpenSci Champions!
```

or something else more "active"?

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+slug: mentors-champions-program
+tags:
+  - community
+  - help
+  - beginner
+  - welcome
+  - diversity
+  - champions program
+feature: true  
+description: "rOpenSci seeks mentors to support our inaugural cohort of rOpenSci Champions. Could you offer insight and advice to our selected champions? Learn the details and express your interest."
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:14]:
```suggestion
description: "rOpenSci is seeking mentors to support our inaugural cohort of rOpenSci Champions. Could you offer insight and advice to our selected champions? Learn the details and express your interest." 
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  - welcome
+  - diversity
+  - champions program
+feature: true  
+description: "rOpenSci seeks mentors to support our inaugural cohort of rOpenSci Champions. Could you offer insight and advice to our selected champions? Learn the details and express your interest." 
+---
+
+
+
+## rOpenSci seeks mentors to support our inaugural cohort of rOpenSci Champions.
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:19]:
```suggestion
## rOpenSci is seeking mentors to support our inaugural cohort of rOpenSci Champions.
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+
+The [rOpenSci Champions Program](/champions/) is for _people from historically and systematically excluded groups_ who are interested in contributing to rOpenSci and the broader ecosystem of open source and open science communities. After receiving training, mentors will work with Champions to develop their project, which can be one of the following options:
+
+* Create a new package
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:24]:
```suggestion
* Create a new package;
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+* Go through the review process with an R package they already have developed
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:26]:
```suggestion
* Go through the review process with an R package they already have developed;
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+* Serve as a reviewer for the rOpenSci  software peer review system
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:28]:
```suggestion
* Serve as a reviewer for the rOpenSci software peer review system.
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+Mentors may also support Champions with delivering other activities including; giving a talk, writing blog posts, and teaching other community members.
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:30]:
```suggestion
Mentors may also support Champions with delivering other activities including: giving a talk, writing blog posts, and teaching other community members.
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+### Role Description
+
+Mentors will play a vital role in the rOpenSci champions program by selecting the first cohort of Champions and advising and inspiring their mentees. Mentors will also __connect__ their mentees to people, programs, organizations; __recommend__, resources, readings, training, experiences and provide __feedback__ about the mentee's project.
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:34]:
```suggestion
Mentors will play a vital role in the rOpenSci champions program by helping select the [first cohort of Champions](/blog/2022/09/22/launch-champions-program/#how-to-apply) and advising and inspiring their mentees. Mentors will also __connect__ their mentees to people, programs, organizations; __recommend__, resources, readings, training, experiences and provide __feedback__ about the mentee's project.
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+
+We are seeking Mentors who are rOpenSci community members who have:
+
+* Demonstrated an understanding and ability to work openly and collaboratively
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:38]:
```suggestion
* Demonstrated an understanding and ability to work openly and collaboratively;
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+
+* The willingness to guide and teach others
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:40]:
```suggestion
* The willingness to guide and teach others;
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+
+* Previous experience with any of the following: 
+
+  * Developing, maintaining or contributing to any of our packages or the packages we review, or
+  
+  * Participating in the rOpenSci review process, or 
+  
+  * Package development and/or peer code review
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:48]:
```suggestion
  * Package development and/or peer code review.
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+
+* Serve as a reviewer for the rOpenSci  software peer review system
+
+Mentors may also support Champions with delivering other activities including; giving a talk, writing blog posts, and teaching other community members.
+
+### Role Description
+
+Mentors will play a vital role in the rOpenSci champions program by selecting the first cohort of Champions and advising and inspiring their mentees. Mentors will also __connect__ their mentees to people, programs, organizations; __recommend__, resources, readings, training, experiences and provide __feedback__ about the mentee's project.
+
+We are seeking Mentors who are rOpenSci community members who have:
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:36]: how is "rOpenSci community members" defined?

Comment (maelle) [reply content/blog/2022-10-20-call-for-mentors-champions-program/index.md:36]: oh I see, in one of the items below, sorry.

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+Mentors will gain tutoring experience and skills, be acknowledged on the rOpenSci website and social media channels, and make new connections across the champions networks. This is a great way to give back to the rOpenSci community by mentoring our selected champions and helping to shape this program by providing feedback on this pilot year. 
+
+### Expectations
+
+The total time commitment is expected to be approximately 1-2 hours per month from November 2022 - December 2023 to complete the following tentative program timeline.
+
+
+|Date|Event|Detail|Duration|
+|----|-----|------|--------|
+|November, 2022|Select Champions |Review, rank and select 1-2 reviewed projects to mentor|2 h|
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:59]:
```suggestion
|November, 2022|Select Champions |Review, rank and select 1-2 reviewed projects to mentor.|2 h|
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+|December 2022|Kick-off event|Learn about the program, review the program expectations, and address questions. Meet mentees.|1.5 h|
+|February 2023|Training|How to mentor and logistics on the program.|1 h|
+|March 2023|Cohort meeting: Presentation of projects|Champions' 5 minutes presentations of their project (package review or development) and get feedback.|1.5 h|
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:62]:
```suggestion
|March 2023|Cohort meeting: Presentation of projects|Champions give 5 minutes presentations of their project (package review or development) and get feedback.|1.5 h|
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+|August - November 2023|Meeting with mentee|Provide feedback and guidance about mentee project.  Connect with other experts, mentors or community member if necessary. Recommend training, resources for the mentee project. |1 meeting per month 0.5 h|
+|September 2023|Cohort meeting: Reflections on events|Champions will be invited to share takeaways from their events with the cohort.|1.5 h|
+|December 2023|Cohort meeting: Reflections on program|Mentors will be invited to share takeaways from the program and help to define future roles for Champions.|1.5 h
+
+
+### Benefits of being a mentor
+
+By becoming an rOpenSci mentor you will:
+
+* Empower members of our community by sharing your expertise with others;
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:74]: should this item mention "historically and systematically excluded groups"?

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 16]
```diff
@@ -0,0 +1,92 @@
...
+* Be the bridge between your mentee and other community members;
+* Gain exposure, recognition and acknowledgment for your work as a mentor in rOpenSci spaces (website, socal media, events)
+* Help rOpenSci and the wider R community grow;
+* Co-develop this program and other rOpenSci projects by sharing feedback;
+
+Mentors will participate in a training about mentoring and program logistics as part of their onboarding process and have the support of the Community Manager throughout the program.
+
+
+### How to apply
+Please, fill out the [application form](https://airtable.com/shrlrLdQbb4wphosK) ([this template shows the information we will require](/champions/files/mentors_champions_template)) before __Monday Nov 7, 2022__. The application forms have to be completed _in English_.
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:85]:
```suggestion
Please, fill out the [application form](https://airtable.com/shrlrLdQbb4wphosK) ([this template shows the information we will require](/champions/files/mentors_champions_template)) before __Monday November 7, 2022__. The application forms have to be completed _in English_.
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 17]
```diff
@@ -0,0 +1,92 @@
...
+
+
+## Do you have any questions? 
+
+Please, contact our Community Manager by [email](mailto:yabellini@ropensci.org) or [book a 15 minutes meeting with her](https://calendly.com/yabellini-ropensci/15min).
+
+We will have a [Community Call October, 24](/commcalls/oct2022-champions/) to share tips for submitting your Champions Program application and a [Coworking space on November, 1](/events/coworking-2022-11/) to help you with your application to the Champions Program.
```
Comment (maelle) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:92]:
```suggestion
We will have a [Community Call October, 24](/commcalls/oct2022-champions/) to share tips for submitting your Champions Program application and a [Coworking slot on November, 1](/events/coworking-2022-11/) to help you with your application to the Champions Program.
```

**Review by maelle — COMMENTED** [review]
So exciting! The description of the role and time needed is very clear! A few comments

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md (hunk 1 above)
Comment (yabellini) [reply content/blog/2022-10-20-call-for-mentors-champions-program/index.md:2]: .Yes!

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md (hunk 15 above)
Comment (yabellini) [reply content/blog/2022-10-20-call-for-mentors-champions-program/index.md:74]: Yes

Comment (yabellini) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:74]:
```suggestion
* Empower members of historically and systematically excluded groups by sharing your expertise with others; 
```

File: content/champions/files/mentors_champions_template.md [hunk 18]
```diff
@@ -0,0 +1,55 @@
...
+  * Yes, I have developed an R package, but I never sent to CRAN.
+  
+  * Yes, I have developed an R package, and it is published in CRAN.
+  
+  * Yes, I have developed an R package before, and it went through rOpenSci review process.
+  
+If you developed a package before please, link the github/gitlab repo:
+
+
+* Do you have experience doing code review?
```
Comment (yabellini) [inline content/champions/files/mentors_champions_template.md:32]:
```suggestion
* Do you have experience reviewing code? 
```

File: content/champions/files/mentors_champions_template.md [hunk 19]
```diff
@@ -0,0 +1,55 @@
...
+If you developed a package before please, link the github/gitlab repo:
+
+
+* Do you have experience doing code review? 
+
+  * No, I have never review code.
+  
+  * No, I have not review code, but I went through the rOpenSci review process.
+  
+  * Yes, I have review code for rOpenSci or other organization.
```
Comment (yabellini) [inline content/champions/files/mentors_champions_template.md:38]:
```suggestion
  * Yes, I have reviewed code for rOpenSci.
```

File: content/champions/files/mentors_champions_template.md [hunk 20]
```diff
@@ -0,0 +1,55 @@
...
+  * Yes, I have developed an R package, and it is published in CRAN.
+  
+  * Yes, I have developed an R package before, and it went through rOpenSci review process.
+  
+If you developed a package before please, link the github/gitlab repo:
+
+
+* Do you have experience doing code review? 
+
+  * No, I have never review code.
```
Comment (yabellini) [inline content/champions/files/mentors_champions_template.md:34]:
```suggestion
  * No, I have never reviewed code.
```

File: content/champions/files/mentors_champions_template.md [hunk 21]
```diff
@@ -0,0 +1,55 @@
...
+  
+  * No, I have not review code, but I went through the rOpenSci review process.
```
Comment (yabellini) [inline content/champions/files/mentors_champions_template.md:36]:
```suggestion
  * No, I have not to reviewed code, but I went through the rOpenSci review process.
```

File: content/champions/files/mentors_champions_template.md [hunk 22]
```diff
@@ -0,0 +1,55 @@
...
+  
+  * Yes, I have review code for rOpenSci or other organization.
+  
+  * Yes, I have been editor for rOpenSci.
+
+
+## Mentoring information
+
+* Would you prefer to mentor a person that: _Choose all that apply_
```
Comment (yabellini) [inline content/champions/files/mentors_champions_template.md:45]:
```suggestion
* Would you prefer to mentor a person that wants to: _Choose all that apply_
```

File: content/champions/files/mentors_champions_template.md [hunk 23]
```diff
@@ -0,0 +1,55 @@
...
+
+  * Wants to develop a new package
```
Comment (yabellini) [inline content/champions/files/mentors_champions_template.md:47]:
```suggestion
  * Develop a new package
```

File: content/champions/files/mentors_champions_template.md [hunk 24]
```diff
@@ -0,0 +1,55 @@
...
+
+  * Wants to send their package to rOpenSci’s review process
```
Comment (yabellini) [inline content/champions/files/mentors_champions_template.md:49]:
```suggestion
  * Send their package to rOpenSci’s review process
```

File: content/champions/files/mentors_champions_template.md [hunk 25]
```diff
@@ -0,0 +1,55 @@
...
+  
+  * Wants to be a reviewer for rOpenSci
```
Comment (yabellini) [inline content/champions/files/mentors_champions_template.md:51]:
```suggestion
  * Be a reviewer for rOpenSci
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 26]
```diff
@@ -0,0 +1,92 @@
...
+slug: mentors-champions-program
+tags:
+  - community
+  - help
+  - beginner
+  - welcome
+  - diversity
+  - champions program
+feature: true  
+description: "rOpenSci is seeking mentors to support our inaugural cohort of rOpenSci Champions. Could you offer insight and advice to our selected champions? Learn the details and express your interest."
```
Comment (steffilazerte) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:14]:
```suggestion
description: "rOpenSci is seeking mentors to support our inaugural cohort of rOpenSci Champions. Could you offer insight and advice to our selected Champions? Learn the details and express your interest!" 
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 27]
```diff
@@ -0,0 +1,92 @@
...
+
+
+### Benefits of being a mentor
+
+By becoming an rOpenSci mentor you will:
+
+* Empower members of historically and systematically excluded groups by sharing your expertise with others; 
+* Connect with other mentors and with the champion community, expanding your professional network;
+* Be the bridge between your mentee and other community members;
+* Gain exposure, recognition and acknowledgment for your work as a mentor in rOpenSci spaces (website, socal media, events)
```
Comment (steffilazerte) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:77]:
```suggestion
* Gain exposure, recognition and acknowledgment for your work as a mentor in rOpenSci spaces (website, social media, events)
```

**Review by steffilazerte — COMMENTED** [review]
Hi @yabellini, so exciting! I can't wait for the program to start!

Most of my suggestions are minor tweaks, and there's one for the airtable form and a couple for content (but still relatively minor).

It's really fun to see how this project is progressing. I can tell that there's a lot of stuff going on behind the scenes :grin:

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 28]
```diff
@@ -0,0 +1,92 @@
...
+  - welcome
+  - diversity
+  - champions program
+feature: true  
+description: "rOpenSci is seeking mentors to support our inaugural cohort of rOpenSci Champions. Could you offer insight and advice to our selected champions? Learn the details and express your interest." 
+---
+
+
+
+## rOpenSci is seeking mentors to support our inaugural cohort of rOpenSci Champions.
```
Comment (yabellini) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:19]:
```suggestion
## rOpenSci is seeking mentors to support our inaugural cohort of rOpenSci Champions!
```

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 29]
```diff
@@ -0,0 +1,92 @@
...
+
+The [rOpenSci Champions Program](/champions/) is for _people from historically and systematically excluded groups_ who are interested in contributing to rOpenSci and the broader ecosystem of open source and open science communities. After receiving training, mentors will work with Champions to develop their project, which can be one of the following options:
+
+* Create a new package;
+
+* Go through the review process with an R package they already have developed;
+
+* Serve as a reviewer for the rOpenSci software peer review system.
+
+Mentors may also support Champions with delivering other activities including: giving a talk, writing blog posts, and teaching other community members.
```
[the remark this answers is not in the document]
Comment (yabellini) [reply content/blog/2022-10-20-call-for-mentors-champions-program/index.md:30]: We expect they will help with the main project but not with all the outreach activities (this is probably more my job). But mentors can advise on events to participate in or topics to write/ prepare a talk, for example.  That is why the _may_ word.

Is that make sense?

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 30]
```diff
@@ -0,0 +1,92 @@
...
+
+* The willingness to guide and teach others;
+
+* Previous experience with any of the following: 
+
+  * Developing, maintaining or contributing to any of our packages or the packages we review, or
+  
+  * Participating in the rOpenSci review process, or 
+  
+  * Package development and/or peer code review.
```
[the remark this answers is not in the document]
Comment (yabellini) [reply content/blog/2022-10-20-call-for-mentors-champions-program/index.md:48]: I think we are open to people from other communities, especially developing a package option.  I think is hard to ask someone that doesn't know our review process to mentor on that topic.  So, how can we make this clear?

A rOpenSci member is anyone that contributes to our community in any way.

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 31]
```diff
@@ -0,0 +1,92 @@
...
+|December 2023|Cohort meeting: Reflections on program|Mentors will be invited to share takeaways from the program and help to define future roles for Champions.|1.5 h
+
+
+### Benefits of being a mentor
+
+By becoming an rOpenSci mentor you will:
+
+* Empower members of historically and systematically excluded groups by sharing your expertise with others; 
+* Connect with other mentors and with the champion community, expanding your professional network;
+* Be the bridge between your mentee and other community members;
```
[the remark this answers is not in the document]
Comment (yabellini) [reply content/blog/2022-10-20-call-for-mentors-champions-program/index.md]: Oh, that is my bias.  I love to be the bridge between people, and I think learning to do that is a skill. 
I like what you add instead.

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 32]
```diff
@@ -0,0 +1,92 @@
...
+
+* Previous experience with any of the following: 
+
+  * Developing, maintaining or contributing to any of our packages or the packages we review, or
+  
+  * Participating in the rOpenSci review process, or 
+  
+  * Package development and/or peer code review.
+
+Mentors will gain tutoring experience and skills, be acknowledged on the rOpenSci website and social media channels, and make new connections across the champions networks. This is a great way to give back to the rOpenSci community by mentoring our selected champions and helping to shape this program by providing feedback on this pilot year.
```
[the remark this answers is not in the document]
Comment (yabellini) [reply content/blog/2022-10-20-call-for-mentors-champions-program/index.md]: It is a good question. Let's see how the text reads with the changes and we can remove it if it is still redundant.

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 33]
```diff
@@ -0,0 +1,92 @@
...
+
+### How to apply
+Please, fill out the [application form](https://airtable.com/shrlrLdQbb4wphosK) ([this template shows the information we will require](/champions/files/mentors_champions_template)) by __Monday November 7, 2022__. The application forms have to be completed _in English_.
+
+
+## Do you have any questions? 
+
+Please, contact our Community Manager by [email](mailto:yabellini@ropensci.org) or [book a 15 minutes meeting with her](https://calendly.com/yabellini-ropensci/15min).
+
+We will have a [Community Call on October 24th](/commcalls/oct2022-champions/) to share tips for submitting your Champions Program application and a [Coworking slot on November 1st](/events/coworking-2022-11/) to help you with your application to the Champions Program.
```
Comment (yabellini) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:92]:
```suggestion
We will have a [Community Call on October 24th](/commcalls/oct2022-champions/)to explain and explore the benefits of Champions Programs and a [Coworking slot on November 1st](/events/coworking-2022-11/) to answer any questions you might have about mentoring in this program.
```

Comment (yabellini) [reply content/blog/2022-10-20-call-for-mentors-champions-program/index.md:92]: as suggested by Steffi.

**Review by yabellini — COMMENTED** [review]
Some other changes :-)

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md (hunk 31 above)
[the remark this answers is not in the document]
Comment (steffilazerte) [reply content/blog/2022-10-20-call-for-mentors-champions-program/index.md]: You could leave it in and just rephrase it as you said, a skill you'll develop! Something like:  "Learn how to be a bridge between a mentee and other community members"

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md (hunk 29 above)
[the remark this answers is not in the document]
Comment (steffilazerte) [reply content/blog/2022-10-20-call-for-mentors-champions-program/index.md:30]: Yup, definitely! "may" makes perfect sense in this context

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md (hunk 30 above)
[the remark this answers is not in the document]
Comment (steffilazerte) [reply content/blog/2022-10-20-call-for-mentors-champions-program/index.md:48]: Perhaps simply add a question to the AirTable form: 
- "How do you participate in the rOpenSci community?" (please answer in 1 or 2 sentences), or,
- "Do you identify as part of the rOpenSci community? How do you participate?"
- 
Then people are free to self-identify and we can omit mentors who say "I don't" :grin: as opposed to answers like "I read the rOpenSci blog and follow rOpenSci on Twitter", or "I use lots of rOpenSci packages in my work", etc.

That way we just leave this list as is and vet whether or not someone is a community member by how they respond when signing up?

**Comment by steffilazerte** [comment]
I just added some comments to your comments/questions, particularly around rOpenSci community membership. I hope they're findable! Let me know if they got lost :grin:

File: content/blog/2022-10-20-call-for-mentors-champions-program/index.md [hunk 34]
```diff
@@ -0,0 +1,90 @@
...
+
+We are seeking mentors from the rOpenSci community who have:
+
+* Demonstrated an understanding and ability to work openly and collaboratively;
+
+* The willingness to guide and teach others;
+
+* Previous experience with any of the following: 
+
+  * Developing, maintaining or contributing to any rOpenSci package or package which have gone through rOpenSci review, or
```
Comment (steffilazerte) [inline content/blog/2022-10-20-call-for-mentors-champions-program/index.md:42]:
```suggestion
  * Developing, maintaining or contributing to any rOpenSci package or package which has gone through rOpenSci review, or
```

**Comment by yabellini** [comment]
The build was ok, and I merged it, but the blog post doesn't appear on the website. I hope you can check what happened @maelle

**Comment by yabellini** [comment]
Can it be related to the feature: true tag?

**Pull request merged**